aggregator/rest_server_test.go

```diff
@@ -18,7 +18,7 @@ func TestGetStateRootUpdateAggregation(t *testing.T) {
 	mockCtrl := gomock.NewController(t)
 	defer mockCtrl.Finish()
 
-	aggregator, _, _, _, _, _, mockDb, _, _, err := createMockAggregator(mockCtrl, MOCK_OPERATOR_PUBKEY_DICT)
+	aggregator, _, _, _, _, _, _, mockDb, _, _, err := createMockAggregator(mockCtrl, MOCK_OPERATOR_PUBKEY_DICT)
```

Comment:
Small comment not for this PR - seems like it's time to export these returns in a neater way. Before we had only a few returns, but now it's quite a few mocks and `_, `, we should just organize this in a struct.

Reply:
Added as an issue in #270